getting_started/setup_vm/roles/ccf_build/vars/clang11.yml

```diff
@@ -0,0 +1,40 @@
+workspace: "/tmp/"
+clang_ver: 11
+
+debs:
+  - apt-transport-https
+  - ninja-build
+  - libuv1-dev
+  - libc++-{{ clang_ver }}-dev
+  - libc++abi-{{ clang_ver }}-dev
+  - python3.8-dev
+  - python3.8-venv
+  - llvm-{{ clang_ver }}
+  - clang-{{ clang_ver }}
+  - clang-format-10 # On purpose, to avoid formatting conflicts
```

Comment:
Ugh. We'll probably have to bite the bullet and do a reformat with 11, we don't want to stay stuck on format-10 forever.

Reply:
#5181